nautobot/dcim/tests/test_views.py

```diff
@@ -2230,6 +2237,9 @@ def setUpTestData(cls):
             "software_version": software_versions[2].pk,
         }
 
+    def test_table_with_indentation_is_removed_on_filter_or_sort(self):
+        self.skipTest("InventroyItem table has no implementation of indentation.")
```

Comment:
```suggestion
        self.skipTest("InventoryItem table has no implementation of indentation.")
```

Though maybe it should??